app/controllers/HomeController.java

```diff
@@ -1,21 +1,58 @@
 package controllers;
 
+import LunchManCore.Apprentice;
+import LunchManCore.FridayLunch;
+import LunchManCore.Rota;
 import play.mvc.*;
 
+import services.CSVHelper;
 import views.html.*;
 
-import java.util.Arrays;
+import java.time.LocalDate;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+
+import static services.CSVHelper.createScheduleFromCSV;
 
 /**
  * This controller contains an action to handle HTTP requests
  * to the application's home page.
  */
 public class HomeController extends Controller {
 
+    private String apprenticeCSV = "/Users/molliestephenson/Java/LunchMan/csvs/apprentices.csv";
+    private String scheduleCSV = "/Users/molliestephenson/Java/LunchMan/csvs/schedule.csv";
+
+    private Rota rota = new Rota(4, LocalDate.now());
+
+    public HomeController() {}
+
+    public HomeController(String apprenticeCSV, String scheduleCSV) {
+        this.apprenticeCSV = apprenticeCSV;
+        this.scheduleCSV = scheduleCSV;
+    }
+
     public Result index() {
-        List<String> apprentices = Arrays.asList("Priya", "Mollie", "Nick", "Rabea", "Ced");
-        return ok(index.render("LunchMan", apprentices));
+        List<Apprentice> apprentices = new ArrayList<>();
+        try {
+            CSVHelper.createApprenticesFromCSV(apprentices, apprenticeCSV);
+            List<FridayLunch> loadedSchedule = createScheduleFromCSV(new ArrayList<>(), scheduleCSV);
+            rota.setSchedule(loadedSchedule);
+            rota.updateSchedule(apprentices);
+            CSVHelper.saveRotaToCSV(rota.getSchedule(), scheduleCSV);
+        } catch (Exception ex) {
+            ex.printStackTrace();
```

Comment:
Try to avoid just printing stack traces out in the catch. 
Its often useful to wrap the original exception in a custom exception to give context.